[2022-06-17T17:11:47.280534+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/"" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/\" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /home/addo/project1/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:186)"} []
[2022-06-17T17:11:47.392402+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T17:11:47.395363+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T17:11:47.811738+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"2f356e"},"request_uri":"http://127.0.0.1:8000/_wdt/2f356e","method":"GET"} []
[2022-06-17T17:11:49.063755+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/favicon.ico" (from "http://127.0.0.1:8000/")" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/favicon.ico\" (from \"http://127.0.0.1:8000/\") at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /home/addo/project1/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2022-06-17T17:11:49.272819+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T17:11:49.273690+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T17:11:49.277146+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T17:11:49.277375+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:11:51.810264+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\ControllerLuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:11:51.834971+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:11:51.840971+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/lucky/number" is not callable: Controller "App\ControllerLuckyController" does neither exist as service nor as class." at /home/addo/project1/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/lucky/number\" is not callable: Controller \"App\\ControllerLuckyController\" does neither exist as service nor as class. at /home/addo/project1/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"App\\ControllerLuckyController\" does neither exist as service nor as class. at /home/addo/project1/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php:67)\n[previous exception] [object] (Error(code: 0): Class 'App\\ControllerLuckyController' not found at /home/addo/project1/vendor/symfony/http-kernel/Controller/ControllerResolver.php:147)"} []
[2022-06-17T18:11:51.899525+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:11:51.899989+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:11:51.901836+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:11:51.902074+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:11:52.481964+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"08dee8"},"request_uri":"http://127.0.0.1:8000/_wdt/08dee8","method":"GET"} []
[2022-06-17T18:12:48.834009+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:12:48.837280+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:12:48.856145+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:12:48.856760+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:12:49.115932+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"67f3a6"},"request_uri":"http://127.0.0.1:8000/_wdt/67f3a6","method":"GET"} []
[2022-06-17T18:12:53.186930+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:12:53.188698+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:12:53.199330+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:12:53.199928+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:12:53.395174+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f69566"},"request_uri":"http://127.0.0.1:8000/_wdt/f69566","method":"GET"} []
[2022-06-17T18:59:40.679165+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:59:40.681044+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:59:40.690158+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:59:40.690648+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:59:41.032935+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a568b8"},"request_uri":"http://127.0.0.1:8000/_wdt/a568b8","method":"GET"} []
[2022-06-17T18:59:43.913418+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:59:43.914679+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:59:43.922283+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:59:43.922700+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:59:44.151139+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"db6204"},"request_uri":"http://127.0.0.1:8000/_wdt/db6204","method":"GET"} []
[2022-06-17T18:59:45.031914+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:59:45.033585+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:59:45.045930+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:59:45.046301+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:59:45.324817+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"7fccaa"},"request_uri":"http://127.0.0.1:8000/_wdt/7fccaa","method":"GET"} []
[2022-06-17T18:59:47.721202+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:59:47.723285+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:59:47.736669+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:59:47.737366+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:59:47.944350+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"6d31a8"},"request_uri":"http://127.0.0.1:8000/_wdt/6d31a8","method":"GET"} []
[2022-06-17T18:59:48.976678+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:59:48.978365+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:59:48.989481+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:59:48.990068+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:59:49.205137+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f36857"},"request_uri":"http://127.0.0.1:8000/_wdt/f36857","method":"GET"} []
[2022-06-17T18:59:49.964719+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:59:49.966239+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:59:49.973888+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:59:49.974279+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:59:50.155593+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"47f8f2"},"request_uri":"http://127.0.0.1:8000/_wdt/47f8f2","method":"GET"} []
[2022-06-17T18:59:50.889131+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:59:50.890779+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:59:50.903905+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:59:50.904500+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:59:51.118459+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"8de330"},"request_uri":"http://127.0.0.1:8000/_wdt/8de330","method":"GET"} []
[2022-06-17T18:59:51.803267+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:59:51.804931+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:59:51.816486+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:59:51.817022+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:59:52.019222+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"72b17d"},"request_uri":"http://127.0.0.1:8000/_wdt/72b17d","method":"GET"} []
[2022-06-17T18:59:52.745182+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T18:59:52.747296+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T18:59:52.756858+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T18:59:52.757373+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T18:59:52.967053+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"453b84"},"request_uri":"http://127.0.0.1:8000/_wdt/453b84","method":"GET"} []
[2022-06-17T20:34:48.277765+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\LuckyController::number"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T20:34:48.290742+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T20:34:48.302138+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:34:48.302640+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:34:48.592865+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"d29042"},"request_uri":"http://127.0.0.1:8000/_wdt/d29042","method":"GET"} []
[2022-06-17T20:45:57.557823+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/lucky/number","method":"GET"} []
[2022-06-17T20:45:57.559189+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T20:45:57.560778+00:00] php.CRITICAL: Fatal Error: Cannot declare class App\Controller\LuckyController, because the name is already in use {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Error: Cannot declare class App\\Controller\\LuckyController, because the name is already in use at /home/addo/project1/src/Controller/DefaultController.php:7)"} []
[2022-06-17T20:45:57.566427+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\FatalError: "Error: Cannot declare class App\Controller\LuckyController, because the name is already in use" at /home/addo/project1/src/Controller/DefaultController.php line 7 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Error: Cannot declare class App\\Controller\\LuckyController, because the name is already in use at /home/addo/project1/src/Controller/DefaultController.php:7)"} []
[2022-06-17T20:45:57.577116+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:45:57.577547+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:45:57.579547+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:45:57.579786+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:45:58.006632+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a0994b"},"request_uri":"http://127.0.0.1:8000/_wdt/a0994b","method":"GET"} []
[2022-06-17T20:46:06.629594+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/default/number"" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/default/number\" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/default/number/\". at /home/addo/project1/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2022-06-17T20:46:06.711405+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:46:06.711915+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:46:06.714435+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:46:06.714682+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:46:07.021931+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"62cd01"},"request_uri":"http://127.0.0.1:8000/_wdt/62cd01","method":"GET"} []
[2022-06-17T20:46:11.929427+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/default/index"" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/default/index\" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/default/index/\". at /home/addo/project1/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2022-06-17T20:46:11.991274+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:46:11.991710+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:46:11.993696+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:46:11.993919+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:46:12.289006+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a76a91"},"request_uri":"http://127.0.0.1:8000/_wdt/a76a91","method":"GET"} []
[2022-06-17T20:46:34.114219+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/default/index"" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/default/index\" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/default/index/\". at /home/addo/project1/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2022-06-17T20:46:34.163524+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:46:34.163932+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:46:34.165741+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:46:34.165982+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:46:34.522897+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"5d97ea"},"request_uri":"http://127.0.0.1:8000/_wdt/5d97ea","method":"GET"} []
[2022-06-17T20:47:07.680670+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/default/index","method":"GET"} []
[2022-06-17T20:47:07.684461+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T20:47:07.700943+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:47:07.701350+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:47:08.057708+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"feb2a6"},"request_uri":"http://127.0.0.1:8000/_wdt/feb2a6","method":"GET"} []
[2022-06-17T20:49:08.181297+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/"" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/\" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /home/addo/project1/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:186)"} []
[2022-06-17T20:49:08.188131+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:49:08.188768+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:49:08.497781+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"019aaa"},"request_uri":"http://127.0.0.1:8000/_wdt/019aaa","method":"GET"} []
[2022-06-17T20:49:17.052597+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/default/index","method":"GET"} []
[2022-06-17T20:49:17.054556+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T20:49:17.066405+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T20:49:17.066945+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T20:49:17.280221+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"5b1577"},"request_uri":"http://127.0.0.1:8000/_wdt/5b1577","method":"GET"} []
[2022-06-17T21:39:27.448411+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/"" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php line 135 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/\" at /home/addo/project1/vendor/symfony/http-kernel/EventListener/RouterListener.php:135)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /home/addo/project1/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:186)"} []
[2022-06-17T21:39:27.567285+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:39:27.570408+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:39:27.930464+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f73401"},"request_uri":"http://127.0.0.1:8000/_wdt/f73401","method":"GET"} []
[2022-06-17T21:40:21.837927+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/default/index","method":"GET"} []
[2022-06-17T21:40:21.846444+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T21:40:21.860489+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:40:21.861070+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:40:22.113393+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a96b24"},"request_uri":"http://127.0.0.1:8000/_wdt/a96b24","method":"GET"} []
[2022-06-17T21:46:46.392499+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/default/index","method":"GET"} []
[2022-06-17T21:46:46.393887+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T21:46:46.526236+00:00] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Variable "cover" does not exist." at /home/addo/project1/templates/home/intro.html.twig line 1 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): Variable \"cover\" does not exist. at /home/addo/project1/templates/home/intro.html.twig:1)"} []
[2022-06-17T21:46:46.596300+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:46:46.596942+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:46:46.599327+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:46:46.599681+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:46:47.115532+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"310f59"},"request_uri":"http://127.0.0.1:8000/_wdt/310f59","method":"GET"} []
[2022-06-17T21:48:05.121343+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/default/index","method":"GET"} []
[2022-06-17T21:48:05.124539+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T21:48:05.141365+00:00] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Variable "cover" does not exist." at /home/addo/project1/templates/home/intro.html.twig line 1 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): Variable \"cover\" does not exist. at /home/addo/project1/templates/home/intro.html.twig:1)"} []
[2022-06-17T21:48:05.234443+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:48:05.235170+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:48:05.238976+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:48:05.239302+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:48:05.808212+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"88e2d2"},"request_uri":"http://127.0.0.1:8000/_wdt/88e2d2","method":"GET"} []
[2022-06-17T21:48:18.394929+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/default/index","method":"GET"} []
[2022-06-17T21:48:18.397538+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T21:48:18.406447+00:00] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Variable "cover" does not exist." at /home/addo/project1/templates/home/intro.html.twig line 1 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): Variable \"cover\" does not exist. at /home/addo/project1/templates/home/intro.html.twig:1)"} []
[2022-06-17T21:48:18.491879+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:48:18.492393+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:48:18.495858+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:48:18.496170+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:48:18.976464+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"afae84"},"request_uri":"http://127.0.0.1:8000/_wdt/afae84","method":"GET"} []
[2022-06-17T21:50:15.302383+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/default/index","method":"GET"} []
[2022-06-17T21:50:15.304690+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T21:50:15.310569+00:00] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Variable "title" does not exist." at /home/addo/project1/templates/home/intro.html.twig line 1 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): Variable \"title\" does not exist. at /home/addo/project1/templates/home/intro.html.twig:1)"} []
[2022-06-17T21:50:15.397936+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:50:15.398973+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:50:15.402139+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:50:15.402450+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:50:15.968070+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f79cf3"},"request_uri":"http://127.0.0.1:8000/_wdt/f79cf3","method":"GET"} []
[2022-06-17T21:50:41.427436+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/default/index","method":"GET"} []
[2022-06-17T21:50:41.429230+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T21:50:41.437203+00:00] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Variable "description" does not exist." at /home/addo/project1/templates/home/intro.html.twig line 8 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): Variable \"description\" does not exist. at /home/addo/project1/templates/home/intro.html.twig:8)"} []
[2022-06-17T21:50:41.501920+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:50:41.503212+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:50:41.505239+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:50:41.505465+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:50:42.063011+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"2cd6bb"},"request_uri":"http://127.0.0.1:8000/_wdt/2cd6bb","method":"GET"} []
[2022-06-17T21:54:06.026624+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/default/index","method":"GET"} []
[2022-06-17T21:54:06.028287+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T21:54:06.035675+00:00] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Variable "figcaption" does not exist." at /home/addo/project1/templates/home/intro.html.twig line 11 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): Variable \"figcaption\" does not exist. at /home/addo/project1/templates/home/intro.html.twig:11)"} []
[2022-06-17T21:54:06.118102+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:54:06.118745+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:54:06.122557+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:54:06.122853+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:54:06.695343+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9ce2c3"},"request_uri":"http://127.0.0.1:8000/_wdt/9ce2c3","method":"GET"} []
[2022-06-17T21:54:34.184154+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/default/index","method":"GET"} []
[2022-06-17T21:54:34.185819+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T21:54:34.191650+00:00] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Variable "content" does not exist." at /home/addo/project1/templates/home/intro.html.twig line 12 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): Variable \"content\" does not exist. at /home/addo/project1/templates/home/intro.html.twig:12)"} []
[2022-06-17T21:54:34.260632+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:54:34.261096+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:54:34.263278+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:54:34.263535+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2022-06-17T21:54:34.729014+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"beb9b0"},"request_uri":"http://127.0.0.1:8000/_wdt/beb9b0","method":"GET"} []
[2022-06-17T21:55:08.830994+00:00] request.INFO: Matched route "app_lucky_number". {"route":"app_lucky_number","route_parameters":{"_route":"app_lucky_number","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://127.0.0.1:8000/default/index","method":"GET"} []
[2022-06-17T21:55:08.833708+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2022-06-17T21:55:08.845539+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"symfony","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"13","charset":"utf8"}} []
[2022-06-17T21:55:08.846239+00:00] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:30)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/addo/project1/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
